Error: Cannot find module 'commander'

## Description
Step 2 fails to unpack the colony-starter-basic package. Following exception is thrown.

```
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'commander'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/andy/.config/yarn/global/node_modules/@colony/colony-starter/index.js:4:19)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

## Steps to Reproduce

0. Meet prerequisites listed on [Getting Started](https://github.com/JoinColony/colonyStarter).
1. Execute step 1.
2. As part of step 2, run following command:
```
colony-starter colony-starter-basic
```

--- a/package.json
+++ b/package.json
@@ -16,6 +16,7 @@
   },
   "devDependencies": {
     "chalk": "^2.4.1",
+    "fs-extra": "^7.0.0",
     "ganache-cli": "^6.1.0",
     "lerna": "^3.0.0-rc.0",
     "trufflepig": "^1.0.4"

--- a/packages/colony-starter/package.json
+++ b/packages/colony-starter/package.json
@@ -14,6 +14,10 @@
   },
   "dependencies": {
     "chalk": "^2.4.1",
-    "tar-pack": "^3.4.1"
+    "commander": "^2.17.1",
+    "cross-spawn": "^6.0.5",
+    "fs-extra": "^7.0.0",
+    "tar-pack": "^3.4.1",
+    "tmp": "^0.0.33"
   }
 }

